pkgs/top-level/all-packages.nix

```diff
@@ -22463,7 +22463,7 @@ with pkgs;
 
   libcacard = callPackage ../development/libraries/libcacard { };
 
-  libcamera = callPackage ../development/libraries/libcamera { };
+  libcamera-qcam = callPackage ../by-name/li/libcamera/package.nix { withQcam = true; };
```

Comment:
Oh the `pkgs/by-name` check fails in a way I didn't really intend to. This specifically should be allowed according to RFC 140: https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md#package-variants

Sorry for that, I'll need to fix this. You can ignore the CI failure!

Reply:
Thanks for the context, and for working on this!